okhttp/src/main/java/okhttp3/Response.java

```diff
@@ -225,10 +225,16 @@ public boolean isRedirect() {
   }
 
   /**
-   * Returns the authorization challenges appropriate for this response's code. If the response code
-   * is 401 unauthorized, this returns the "WWW-Authenticate" challenges. If the response code is
-   * 407 proxy unauthorized, this returns the "Proxy-Authenticate" challenges. Otherwise this
-   * returns an empty list of challenges.
+   * Returns the RFC 7235 authorization challenges appropriate for this response's code.
+   * If the response code is 401 unauthorized, this returns the "WWW-Authenticate" challenges.
+   * If the response code is 407 proxy unauthorized, this returns the "Proxy-Authenticate"
+   * challenges. Otherwise this returns an empty list of challenges.
+   * <p>
+   * If a challenge uses the {@code token68} variant instead of auth params,
+   * there is exactly one auth param in the challenge at key {@code null}.
```

Comment:
key null doesn’t feel great, though I don’t have better ideas

Reply:
Well, I had in mind either the token as key and a null value or a null key and the token as value or a separate method.
But as it is an exclusive or between the token and the auth params I thought this is the most elegant solution.
If you want me to change it to something else, or maybe a fixed key `token68`, just tell me.